katago_server/startposes/models/startpos.py

```diff
@@ -64,3 +65,17 @@ class Meta:
     cumulative_weight = FloatField(
         _("cumulative_weight"), default=-1, null=False, help_text=_("Cumulative weight, for efficient random selection."), db_index=True,
     )
+
+class StartPosCumWeightOnly(Model):
```

Comment:
I find the naming confusing: `SimplerStartPositionForBulkUpdate`

Reply:
Maybe. It's specifically for updating cumulative weight though, if we wanted to bulk update a different field then we'd need to make a second, separate unmanaged model.